src/interface.js

```diff
@@ -64,11 +64,11 @@ export default function(Target) {
   // "then" method on the current `Target`
   each(['bind', 'catch', 'finally', 'asCallback',
     'spread', 'map', 'reduce', 'tap', 'thenReturn',
-    'return', 'yield', 'ensure', 'reflect'], function(method) {
+    'return', 'yield', 'ensure', 'reflect',
+    'get', 'mapSeries', 'delay'], function(method) {
     Target.prototype[method] = function() {
-      let then = this.then();
-      then = then[method].apply(then, arguments);
-      return then;
+      const promise = this.then(identity);
```

Comment:
It doesn't need to be. Before I realize that `.then` doesn't need a parameter, in order to use bluebird's `.get`, I've been using `.then(_.identity).get(0)`. I've pushed a commit to remove it.